frontend/src/host_orchestrator/main.go

```diff
@@ -53,6 +55,34 @@ func startHttpServer(addr string, port int) error {
 	return http.ListenAndServe(fmt.Sprintf("%s:%d", addr, port), nil)
 }
 
+// Logs the version of relevant HO service dependencies.
+func logDepsVersions() {
+	var deps = []struct {
+		name string
+		cmd  string
+		args []string
+	}{
+		{
+			name: "kernel",
+			cmd:  "uname",
+			args: []string{"-r"},
+		},
+		{
+			name: "cvd",
+			cmd:  "cvd",
+			args: []string{"version"},
```

Comment:
Newer cvds print the version on every command invocation as long as stdio is not an interactive terminal.

Reply:
Good to know, I wasn't aware of it. Let's have it printed at the beginning of the logs for discover-ability purposes as well.